lib/misc.c

```diff
@@ -162,6 +162,13 @@ void strip_html(char *in)
 
 	memset(out, 0, sizeof(out));
 
+	if (strlen(in) == 0) {
```

Comment:
That works too. For my own morbid curiosity because C is FAR from my best language - what's the benefit to one over the other?